zeppelin-interpreter/pom.xml

```diff
@@ -213,6 +213,13 @@
       <exclusions>
       </exclusions>
     </dependency>
+
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scala-library</artifactId>
+      <version>${scala.version}</version>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
Because `BaseZeppelinContext` use scala api, and to make it extensible by other interpreters, I move it from `zeppelin-spark` to `zeppelin-interpreter`.